pom.xml

```diff
@@ -266,6 +266,25 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>com.spotify</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
```

Comment:
I'm so glad you got this figured out! I couldn't, for the life of me, get this plugin to work correctly! :)